tests/vmi_monitoring_test.go

```diff
@@ -70,9 +70,9 @@ var _ = Describe("Health Monitoring", func() {
 			By("Killing the watchdog device")
 			_, err = expecter.ExpectBatch([]expect.Batcher{
 				&expect.BSnd{S: "watchdog -t 2000ms -T 4000ms /dev/watchdog && sleep 5 && killall -9 watchdog\n"},
-				&expect.BExp{R: "#"},
+				&expect.BExp{R: "\\#"},
```

Comment:
we want to add Snd \n + Exp prompt before the first Snd ?

Reply:
Why? And how this is relevant to this PR?

Reply:
it not must be in this PR, 
i just saw it and wrote, as most of the batches start with it , to be independent on the previous as a preliminary act, i think its a good practice, at least in the current state of the code.